src/lib/snack-bar/snack-bar.ts

```diff
@@ -64,6 +62,10 @@ export class MdSnackBar {
     } else {
       snackBarRef.containerInstance.enter();
     }
+    if (config.autoHide > 0) {
```

Comment:
Can we use strict null checking?

We can then use the Non-null assertion operator

for instance in this case it would be:

`if (config.autoHide! > 0) {`